src/modal/component/Modal/Modal.tsx

```diff
@@ -174,10 +162,10 @@ function Modal(): JSX.Element {
                   }
 
                   try {
-                    const ledgerWalletProvider = state.walletProviders[
-                      "ledgerwallet"
-                    ] as ILedgerWallet;
+                    const ledgerWalletProvider = state.walletProviders["ledgerwallet"] as ILedgerWallet;
                     ledgerWalletProvider.setDerivationPath(derivationPath);
+                    ledgerWalletProvider.setAccountId(accountId);
+                    console.log(accountId);
```

Comment:
Do we still need this `console.log`?

Reply:
I think we should remove all console.log from the code base?

Reply:
No, it can be removed.